ports/unix/main.c

```diff
@@ -753,10 +754,7 @@ MP_NOINLINE int main_(int argc, char **argv) {
     #endif
 
     #if MICROPY_PY_SYS_ATEXIT
-    // Beware, the sys.settrace callback should be disabled before running sys.atexit.
```

Comment:
This comment was added in https://github.com/micropython/micropython/commit/310b3d1b81d561e19d719acd89ee47b759e3795c

Does anyone know why this would be the case and if it's still relevant?